mne_bids_pipeline/steps/source/_01_make_bem_surfaces.py

```diff
@@ -154,15 +154,16 @@ def main(*, config: SimpleNamespace) -> None:
             mne.datasets.fetch_fsaverage(get_fs_subjects_dir(config))
         return
 
-    # check for session-specific MRIs within subject, and handle accordingly
+    # check for session-specific MRIs within subject, and add entries to `subj_sess` for
+    # each combination of subject+session that has its own MRI
     subjects_dir = Path(get_fs_subjects_dir(config))
-    subj_sess = list()
+    subj_sess = set()
```

Comment:
should remove redundancy, which probably wasn't hurting anything other than possibly cluttering the logs (conceivably could have caused a race condition in creating redundant BEMs for the same subj but different sessions? anyway it shouldn't do that anymore, if it ever did).